exporter.py

```diff
@@ -277,8 +282,8 @@ def translate(self, scene: RamsesPython.Scene, ir_node: Node, exportable_scene:
 
         log.debug(f'Translated IRNode "{str(ir_node)}" into "{returned_node}"')
         # TODO: get RAMSES node name from bindings
-
-        return returned_node
+        last_transformation.addChild(returned_node)
+        return last_transformation
```

Comment:
why not return the node, but instead the last transformation?

Reply:
That is because I am trying to refactor 'translate' to return a list of all transformations applied, i.e. I want a call to translate() to eventually return scaling -> rotation -> translation -> node. This is in line with what I was trying to do in my last pull request. However, maybe 'last_transformation' is a misnomer and I should name it better.